loadbalancer_public.tf

```diff
@@ -145,7 +148,7 @@ resource "aws_lb_target_group" "quortex_public" {
   # this group id in an autoscaling group.
 
   # No target group will be created if the target port is not defined
-  count = length(var.load_balancer_public_app_backend_ports)
+  count = length(var.load_balancer_public_app_backend_ports) > 0 ? length(var.load_balancer_public_app_backend_ports) : 0
```

Comment:
Is this change is needed ? 

If len(var.load_balancer_public_app_backend_ports) == 0, the len(var.load_balancer_public_app_backend_ports) is already equal to 0.